packages/core/src/shared/wizards/wizard.ts

```diff
@@ -269,9 +271,7 @@ export class Wizard<TState extends Partial<Record<keyof TState, unknown>>> {
 
         if (isValidResponse(answer)) {
             state.stepCache.picked = prompter.recentItem
-        }
-
-        if (!isValidResponse(answer)) {
```

Comment:
Are we sure we can change this to an else? Is there a test to verify?

Reply:
This an obvious binary case to me.